### DISCUSSION TITLE
[Question] Single instance

### DISCUSSION THREAD

**Discussion by JoseitoOliveira** [flet-dev/flet#3243] [Q&A] [answered]
### Question

Is it possible to create a single instance Flet web application? I would like to create a simple page just for my local network and run it on a raspberry controlling its GPIOs.

![nRonVUPN](https://github.com/flet-dev/flet/assets/45806413/c9009edd-63c2-4a84-9848-f6f3aacb273c)

As these are the same GPIOs no matter who is accessing the APP, it makes sense that all connections have the same state.
Flet's default behavior is not to share state between different open pages, is it possible to change this?

I asked the same question on stackoverflow https://stackoverflow.com/questions/78449024/python-flet-single-instance

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Try [`PubSub`](https://flet.dev/docs/cookbook/pub-sub).

**Reply by JoseitoOliveira** [reply]
Thanks, it's not the approach I had in mind but the result is the same.
